src/containers/User/UserArea.tsx

```diff
@@ -1,47 +1,73 @@
-import React from 'react'
-import { Button, Grid, Label, Segment, Sticky } from 'semantic-ui-react'
+import React, { useEffect, useState } from 'react'
+import { Button, Grid, Header, Label, Menu, MenuProps, Segment, Sticky } from 'semantic-ui-react'
 import strings from '../../utils/constants'
 import { useUserState } from '../../contexts/UserState'
 import UserSelection from './UserSelection'
 import { useRouter } from '../../utils/hooks/useRouter'
+import useListTemplates from '../../utils/hooks/useListTemplates'
 
 const UserArea: React.FC = () => {
   const {
-    userState: { currentUser },
+    userState: { currentUser, isLoading, templatePermissions },
     logout,
   } = useUserState()
-  const { replace } = useRouter()
+  const {
+    replace,
+    push,
+    query: { type },
+  } = useRouter()
+  const [menuItems, setMenuItems] = useState<MenuProps>()
+  const { error, loading, filteredTemplates } = useListTemplates(templatePermissions, isLoading)
+
+  useEffect(() => {
+    if (filteredTemplates) {
+      const items = filteredTemplates.map(({ name, code, permissions }) => ({
+        key: `app_menu_${name}`,
+        content: <Header as="h5" inverted content={name} />,
+        active: type === code,
+        onClick: () => push(`/applications?type=${code}&user-role=${permissions[0]}`),
+      }))
+      const homeItem = {
+        key: `app_menu_home`,
+        content: <Header as="h5" inverted icon="home" />,
+        active: location.pathname === '/',
+        onClick: () => push('/'),
+      }
+      setMenuItems({ items: [homeItem, ...items] })
```

Comment:
>  Is is using the "shorthand" style.

Yep

Reply:
> And I think it would be better to use the link attribute where possible, so they show up link conventional links -- maybe this will work properly too with the above suggestion?

I'm doing this because I'm setting this Menu items in the `useEffect`, that's why the shorthand option seemed alright, but if it's not so clear... Ok, I had an idea. I will try it.

Reply:
Ah, I think it's not going to be very helpful... Maybe let's continue with using the `shorthand` I think once we get used is a good practice. Helps to preset the items on a component. Also this code should be moved to the AppMenu component at some point so it would be better allocated. 

I'll change to use the **Link** now

Reply:
ok ok, I'll change to use <Menu.Item>, the **Link** is not working with the `shorthand` solution... I guess I can check for {filteredTemplates} on the render.

Reply:
Done: [a1adc13](https://github.com/openmsupply/application-manager-web-app/pull/262/commits/a1adc13548833f0d0b1cdc334884d4d5a3676fcd)